src/org/opendatakit/briefcase/ui/export/components/ConfigurationPanelForm.java

```diff
@@ -327,14 +329,14 @@ private boolean confirmOverwriteFiles() {
     gbc.anchor = GridBagConstraints.EAST;
     container.add(pemFileLabel, gbc);
     startDateLabel = new JLabel();
-    startDateLabel.setText("Start date");
+    startDateLabel.setText("Start date (inclusive)");
     gbc = new GridBagConstraints();
     gbc.gridx = 0;
     gbc.gridy = 2;
     gbc.anchor = GridBagConstraints.EAST;
     container.add(startDateLabel, gbc);
     endDateLabel = new JLabel();
-    endDateLabel.setText("End date");
+    endDateLabel.setText("End date (inclusive)");
```

Comment:
In this file, remove all changes except this one and the previous

Reply:
Which changes are you talking about here?

Reply:
Take a look at the screenshot. You'll notice how there are some changes in the order of the imports and some added empty lines, which shouldn't be merged.

![image](https://user-images.githubusercontent.com/205913/40664207-b06c42da-635a-11e8-8a4d-6ce02dd921d3.png)

![image](https://user-images.githubusercontent.com/205913/40664166-9be44858-635a-11e8-96b1-7223214df6c1.png)